File: src/URI/URI_1052.java
package URI;
import java.util.Scanner;

public class URI_1052 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		
		String[] meses = new String[] {"", "January", "February", "March", "April", "May", "June",
									  "July", "August", "September", "October", "November", "December"};
							
		int n = sc.nextInt();
		System.out.println(meses[n]);				
					
		sc.close();

	}

}

File: kotlin/URI_1052.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)

    val meses = arrayOf(
        "", "January", "February", "March", "April", "May", "June",
        "July", "August", "September", "October", "November", "December"
    )

    val n = sc.nextInt()
    println(meses[n])
}
